models/members.js

```diff
@@ -33,6 +33,31 @@ const fetchMembers = async () => {
   }
 }
 
+/**
+ * Fetches the data about our member usernames
```

Comment:
`data about our member usernames`..... 🤔 

What about something like `Returns usernames of all the members in the database`?